components/ActionSheet/tests/unit/ActionSheetTest.swift

```diff
@@ -90,10 +90,13 @@ class ActionSheetTest: XCTestCase {
     // Then
     XCTAssertEqual(actionSheet.backgroundColor, .white)
     XCTAssertEqual(actionSheet.view.backgroundColor, .white)
-    let subviewsArray = actionSheet.view.subviews
-    for view in subviewsArray {
-      XCTAssertEqual(view.backgroundColor, .white)
-    }
+
+    //  Disable this test because of XCode beta.
+    //  Tracking issue (https://github.com/material-components/material-components-ios/issues/8238)
```

Comment:
Please mark this using TODO notation:

```
// TODO(https://github.com/material-components/material-components-ios/issues/8238): Re-enable this test.
```

Reply:
Addressed in 72283648bacf7efcd28cbd4b0af638582f37ab03